### PR TITLE
first-commit

### PR DESCRIPTION
**Pull request by julioo**
Update documentation to share on KF documentation website

### CODE REVIEW

**Comment by julioo** [comment]
I have read the CLA Document and I hereby sign the CLA

File: README.md [hunk 1]
```diff
@@ -1,29 +1,330 @@
-# kubeflow-oke
+# Kubeflow-Oke
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
# Kubeflow for Oracle Container Engine for Kubernetes
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,330 @@
...
+[uri-kubernetes]: https://kubernetes.io/
+[uri-oci]: https://cloud.oracle.com/cloud-infrastructure
+[uri-oci-documentation]: https://docs.cloud.oracle.com/iaas/Content/home.htm
+[uri-oke]: https://docs.cloud.oracle.com/iaas/Content/ContEng/Concepts/contengoverview.htm
+[uri-oracle]: https://www.oracle.com
 
 [![License: UPL](https://img.shields.io/badge/license-UPL-green)](https://img.shields.io/badge/license-UPL-green) [![Quality gate](https://sonarcloud.io/api/project_badges/quality_gate?project=oracle-devrel_kubeflow-oke)](https://sonarcloud.io/dashboard?id=oracle-devrel_kubeflow-oke)
 
-## THIS IS A NEW, BLANK REPO THAT IS NOT READY FOR USE YET.  PLEASE CHECK BACK SOON!
+<!-- ## THIS IS A NEW, BLANK REPO THAT IS NOT READY FOR USE YET.  PLEASE CHECK BACK SOON! -->
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,29 +1,330 @@
...
+
+## Table of Contents
+
+<!-- toc -->
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,29 +1,330 @@
...
+
+  - [Introduction](#introduction)
+  - [Installation](#installation)
+    - [Prerequisites](#prerequisites)
+    - [Create OKE](#create-oke)
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    - [Create an Oracle Container Engine for Kubernetes (OKE) Cluster](#create-oke)
```

File: README.md [hunk 5]
```diff
@@ -1,29 +1,330 @@
...
+    - [Install Kubeflow](#install-kubeflow)
+    - [Access Kubeflow Dashboard](#access-kubeflow-dashboard)
+  - [Notes/Issues](#notesissues)
+  - [URLs](#urls)
+  - [Contributing](#contributing)
+  - [License](#license)
+
+<!-- tocstop -->
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1,29 +1,330 @@
...
 
 ## Introduction
-MISSING
+
+Running Kubeflow on [Oracle Container Engine (OKE)][uri-oke].
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -1,29 +1,330 @@
...
+
+OKE is [Oracle's][uri-oracle] managed [Kubernetes][uri-kubernetes] service on [Oracle Cloud Infrastructure (OCI)][uri-oci].
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
Oracle Container Engine for Kubernetes (OKE) is the [Oracle][uri-oracle]-managed [Kubernetes][uri-kubernetes] service on [Oracle Cloud Infrastructure (OCI)][uri-oci]. 
```

File: README.md [hunk 8]
```diff
@@ -1,29 +1,330 @@
...
 
 ## Getting Started
-MISSING
+
+> ⚠️ Kubeflow 1.5.0 is not compatible with Kubernetes version 1.22 and onwards. To install Kubeflow 1.5.0 and older versions, use OKE v1.21.5.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
> ⚠️ Kubeflow 1.5.0 is not compatible with Kubernetes version 1.22 and onwards. To install Kubeflow 1.5.0 or older, set the Kubernetes version of your OKE cluster to v1.21.5.
```

File: README.md [hunk 9]
```diff
@@ -1,29 +1,330 @@
...
+
+This guide explains how to install Kubeflow 1.6.0 on OKE 1.22.5, 1.23.4 and 1.24.1.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
This guide explains how to install Kubeflow 1.6.0 on OKE using Kubernetes versions 1.22.5, 1.23.4 and 1.24.1.
```

File: README.md [hunk 10]
```diff
@@ -1,29 +1,330 @@
...
+## Installation
+
+This guide describes how to create a Kubernetes cluster using OKE, access OKE, install Kubeflow and expose Kubeflow Dashboard.
 
 ### Prerequisites
-MISSING
 
-## Notes/Issues
-MISSING
+This guide can be run in many different ways, but in all cases, you will need access to an Oracle Cloud Tenancy and be signed in to it.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
This guide can be run in many different ways, but in all cases, you will need to be signed into an Oracle Cloud Infrastructure tenancy.
```

File: README.md [hunk 11]
```diff
@@ -1,29 +1,330 @@
...
+
+The steps described below are executed from the Cloud Shell in the OCI console.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
This guide uses the [Cloud Shell](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/devcloudshellintro.htm) built into the Oracle Cloud Console but can also be run from your local workstation.
```

File: README.md [hunk 12]
```diff
@@ -1,29 +1,330 @@
...
+
+### Create an OKE cluster
+
+<!-- <details><summary><b>Create an OKE cluster manually</b>
+</summary> -->
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

File: README.md [hunk 13]
```diff
@@ -1,29 +1,330 @@
...
+
+    ![Select Compartment](images/SelectCompartment.png)
+
+3. On the Create Cluster dialog, click **Quick Create** and click **Launch Workflow**.
+
+    ![Launch Workflow](images/LaunchWorkFlow.png)
+
+    *Quick Create* will automatically create a new cluster with the default settings and new network resources for the new cluster.
+
+4. Specify the following configuration details on the Cluster Creation dialog (pay attention to the value you place in the **Shape** field):
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
4. Specify the following configuration details on the Cluster Creation dialog paying close attention to the value you use in the **Shape** field:
```

File: README.md [hunk 14]
```diff
@@ -1,29 +1,330 @@
...
+
+    * **Name**: The name of the cluster. Accept the default value.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    * **Name**: The name of the cluster. Accept the default value or provide your own.
```

File: README.md [hunk 15]
```diff
@@ -1,29 +1,330 @@
...
+    * **Compartment**: The name of the compartment. Accept the default value.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    * **Compartment**: The name of the compartment. Accept the default value or provide your own.
```

File: README.md [hunk 16]
```diff
@@ -1,29 +1,330 @@
...
+    * **Kubernetes Version**: The version of Kubernetes. Select the **v1.23.4** version.
+    * **Kubernetes API Endpoint**: Determines if the cluster master nodes are going to be routable or not. Select the **Public Endpoint** value.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    * **Kubernetes API Endpoint**: Determines if the cluster control plane nodes will be directly accessible from external sources. Select the **Public Endpoint** value.
```

File: README.md [hunk 17]
```diff
@@ -1,29 +1,330 @@
...
+    * **Kubernetes Worker Nodes**: Determines if the cluster worker nodes are going to be routable or not. Accept the default value **Private Workers**.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    * **Kubernetes Worker Nodes**: Determines if the cluster worker nodes will be directly accessible from external sources. Accept the default value **Private Workers**.
```

File: README.md [hunk 18]
```diff
@@ -1,29 +1,330 @@
...
+    * **Shape**: The shape to use for each node in the node pool. The shape determines the number of CPUs and the amount of memory allocated to each node. The list shows only those shapes available in your tenancy that are supported by OKE.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    * **Shape**: The shape to use for each node in the node pool. The shape determines the number of OCPUs and the amount of memory allocated to each node. The list shows only those shapes available in your tenancy that are supported by OKE.
```

File: README.md [hunk 19]
```diff
@@ -1,29 +1,330 @@
...
+    * **Number of nodes**: The number of worker nodes to create. Accept the default value **3**.
+<br>
```
Comment (Djelibeybi) [inline README.md]:
```suggestion

```

File: README.md [hunk 20]
```diff
@@ -1,29 +1,330 @@
...
+    > **Caution**: *VM.Standard.E4.Flex is the recommended shape, but you can adjust based on your requirements and location.*
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    > **Caution**: *VM.Standard.E4.Flex is the recommended shape but is not available to Always Free tenancies. You should adjust the shape based on your requirements and location.*
```

File: README.md [hunk 21]
```diff
@@ -1,29 +1,330 @@
...
+
+  ![Quick Cluster](images/oke-specs.png)
+  <!-- ![Enter Data](images/ClusterShape2.png) -->
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

File: README.md [hunk 22]
```diff
@@ -1,29 +1,330 @@
...
+
+1. Click **Next** to review the details you entered for the new cluster.
+
+2. On the Review page, click **Create Cluster** to create the new network resources and the new cluster.
+
+    <!-- ![Review Cluster](images/CreateCluster.png) -->
+
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

File: README.md [hunk 23]
```diff
@@ -1,29 +1,330 @@
...
+    > You see the network resources being created for you. Wait until the request to create the node pool is initiated and then click **Close**.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    You see the network resources being created for you. Wait until the request to create the node pool is initiated and then click **Close**.
```

File: README.md [hunk 24]
```diff
@@ -1,29 +1,330 @@
...
+
+    ![Network Resource](images/NetworkCreation.png)
+
+    > The new cluster is shown on the Cluster Details page. When the master nodes are created, the new cluster gains a status of *Active* (it takes about 7 minutes).
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    The new cluster is shown on the Cluster Details page. When the control plane nodes are created, the status of new cluster becomes *Active*. This usually takes around 10 minutes.
```

File: README.md [hunk 25]
```diff
@@ -1,29 +1,330 @@
...
+
+    ![cluster1](images/ClusterProvision.png)
+
+    ![cluster1](images/ClusterActive.png)
+
+    Now your OKE cluster is ready. You can move to Kubeflow installation.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    Now that your OKE cluster is ready, we can move on to the Kubeflow installation.
```

File: README.md [hunk 26]
```diff
@@ -1,29 +1,330 @@
...
+
+### Install Kubeflow
+
+# Setup Kubeflow
+
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

File: README.md [hunk 27]
```diff
@@ -1,29 +1,330 @@
...
+    > The new cluster is shown on the Cluster Details page. When the master nodes are created, the new cluster gains a status of *Active* (it takes about 7 minutes).
+
+    ![cluster1](images/ClusterProvision.png)
+
+    ![cluster1](images/ClusterActive.png)
+
+    Now your OKE cluster is ready. You can move to Kubeflow installation.
+
+### Install Kubeflow
+
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
## Install Kubeflow

```

File: README.md [hunk 28]
```diff
@@ -1,29 +1,330 @@
...
+# Setup Kubeflow
+
+## Prerequisites
+
+The following tools are required:
+- oci-cli,
+- kubectl,
+- git are integrated
+
+In this guide, we will use the OCI Cloud Shell console, and all these tools are integrated.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
The following utilities are required to install Kubeflow:

- [OCI-CLI](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/cliconcepts.htm)
- [kubectl](https://kubernetes.io/docs/reference/kubectl/)
- [git](https://git-scm.com/)

These tools are installed by default in your Cloud Shell instance.
```

File: README.md [hunk 29]
```diff
@@ -1,29 +1,330 @@
...
+
+### Warning
+
+<!-- - **Kubeflow 1.6.0 is not compatible with version Kubernetes 1.22 and backwards.** -->
+- Kustomize (version 3.2.0) ([download link](https://github.com/kubernetes-sigs/kustomize/releases/tag/v3.2.0))
+    - Kubeflow 1.6.0 is not compatible with the latest versions of Kustomize.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
> **Warning:** use [Kustomize v3.2.0](https://github.com/kubernetes-sigs/kustomize/releases/tag/v3.2.0) as Kubeflow 1.6.0 is not compatible with the later versions of Kustomize.
```

File: README.md [hunk 30]
```diff
@@ -1,29 +1,330 @@
...
+
+## Preparation steps
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
### Preparation
```

File: README.md [hunk 31]
```diff
@@ -1,29 +1,330 @@
...
+
+1. Install Kustomize in Cloud Shell
+
+    - Open the OCI Cloud Shell
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    - Open Cloud Shell
```

File: README.md [hunk 32]
```diff
@@ -1,29 +1,330 @@
...
+
+    - Make sure you are in the top level directory
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    - Make sure you are in your home directory
```

File: README.md [hunk 33]
```diff
@@ -1,29 +1,330 @@
...
+
+            cd $HOME
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
            cd ~
```

File: README.md [hunk 34]
```diff
@@ -1,29 +1,330 @@
...
+
+    - Download Kustomize version 3.2.0 and install in into your OCI Cloud Shell environment.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
    - Download Kustomize v3.2.0 and install it in your Cloud Shell environment.
```

File: README.md [hunk 35]
```diff
@@ -1,29 +1,330 @@
...
+
+            curl -L -o kustomize https://github.com/kubernetes-sigs/kustomize/releases/download/v3.2.0/kustomize_3.2.0_linux_amd64
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
            curl -sfL https://github.com/kubernetes-sigs/kustomize/releases/download/v3.2.0/kustomize_3.2.0_linux_amd64 -o kustomize
```

File: README.md [hunk 36]
```diff
@@ -1,29 +1,330 @@
...
+            chmod +x ./kustomize
+
+2. Clone the Kubeflow repository version 1.6.0 into your OCI Cloud Shell environment
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
2. Clone the v1.6.0 branch of the Kubeflow manifests repository to your Cloud Shell environment
```

File: README.md [hunk 37]
```diff
@@ -1,29 +1,330 @@
...
+
+        git clone -b v1.6-branch https://github.com/kubeflow/manifests.git kubeflow_1.6
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
        git clone -b v1.6-branch https://github.com/kubeflow/manifests.git kubeflow-1.6
```

File: README.md [hunk 38]
```diff
@@ -1,29 +1,330 @@
...
+        cd kubeflow_1.6
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
        cd kubeflow-1.6
```

File: README.md [hunk 39]
```diff
@@ -1,29 +1,330 @@
...
+        PASSWORD=YOURPASSWORD
+        kubeflow_password=$(htpasswd -nbBC 12 USER $PASSWORD| sed -r 's/^.{5}//')
+
+    Edit Dex config-map.yaml
+  
+    Make sure you are in the kubeflow manifests folder.
+
+        cd $HOME/kubeflow_1.6/
+        cp common/dex/base/config-map.yaml{,.org}
+        cat common/dex/base/config-map.yaml.org | sed "s|hash:.*|hash: $kubeflow_password|" > common/dex/base/config-map.yaml
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
3. Replace the default email address and password used by Kubeflow with a randomly generated one:
    
   ```shell
   $ PASSWD=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 55 | head -n 1)
   $ KF_PASSWD=$(htpasswd -nbBC 12 USER $PASSWORD| sed -r 's/^.{5}//')
   $ sed -i.orig "s|hash:.*|hash: $KF_PASSWD|" common/dex/base/config-map.yaml 
   $ echo "Random password is: $PASSWD"
   ```
```

Comment (Djelibeybi) [reply README.md]: The formatting may be a little off on this one, so please double-check!

File: README.md [hunk 40]
```diff
@@ -1,29 +1,330 @@
...
+
+## Install Kubeflow
+
+1. Access your cluster
+
+  Click Access Cluster on your cluster detail page.
+  Accept the default **Cloud Shell Access** and click **Copy** copy the `oci ce cluster create-kubeconfig ...` command and paste it into the Cloud Shell and run the command.
+  <!-- > If you installed your cluster using the single-script the access is already set. -->
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
1. To access your OKE cluster, click **Access Cluster** on the cluster detail page. Accept the default **Cloud Shell Access** and click **Copy** to copy the `oci ce cluster create-kubeconfig ...` command. Next, paste it into your Cloud Shell session and hit **Enter**.
```

File: README.md [hunk 41]
```diff
@@ -1,29 +1,330 @@
...
+
+  ![cluster1](images/AccessCluster.png)
+
+  Verify that the `kubectl` is working by using the `get nodes` command. <br>
+  You may need to run this command several times until you see an output similar to the following.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
  Verify that the `kubectl` is working by using the `get nodes` command. You may need to repeat this command several times until all three nodes show `Ready` in the `STATUS` column.
```

File: README.md [hunk 42]
```diff
@@ -1,29 +1,330 @@
...
+
+  ```bash
+    $ kubectl get nodes
+    NAME          STATUS   ROLES   AGE   VERSION
+    10.0.10.176   Ready    node    19m   v1.23.4
+    10.0.10.203   Ready    node    19m   v1.23.4
+    10.0.10.48    Ready    node    19m   v1.23.4
+  ```
+
+  > If you see the node's information, then the configuration was successful.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
  When all three nodes are `Ready`, your OKE install has finished successfully.
```

File: README.md [hunk 43]
```diff
@@ -1,29 +1,330 @@
...
+
+2. Install Kubeflow with a single command
+
+        cd $HOME/kubeflow_1.6
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
        cd $HOME/kubeflow-1.6 
```

File: README.md [hunk 44]
```diff
@@ -1,29 +1,330 @@
...
+  ```
+
+  > If you see the node's information, then the configuration was successful.
+
+2. Install Kubeflow with a single command
+
+        cd $HOME/kubeflow_1.6 
+        while ! $HOME/kustomize build example | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
+
+> Installation takes about 10 to 15 min.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
Installation usually takes between 10 to 15 minutes.
```

File: README.md [hunk 45]
```diff
@@ -1,29 +1,330 @@
...
+
+To check that all Kubeflow-related Pods are ready, use the following commands:
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
Use the following commands to check if all the pods related to Kubeflow are ready:
```

File: README.md [hunk 46]
```diff
@@ -1,29 +1,330 @@
...
+kubectl get pods -n cert-manager
+kubectl get pods -n istio-system
+kubectl get pods -n auth
+kubectl get pods -n knative-eventing
+kubectl get pods -n knative-serving
+kubectl get pods -n kubeflow
+kubectl get pods -n kubeflow-user-example-com
+ ```
+
+>It takes about 15 min to have all the containers running.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
It usually takes about 15 minutes for all the pods to start successfully.
```

File: README.md [hunk 47]
```diff
@@ -1,29 +1,330 @@
...
+
+### Access Kubeflow Dashboard
+
+#### Expose Kubeflow to Internet
+
+Kubeflow v1.6 already deploy a load balancer and expose the dashboard to Internet.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
## Accessing the Kubeflow Dashboard

To enable access to the Kubeflow Dashboard from the internet, we first need to change the default `istio-ingressgateway` service created by Kubeflow into a `LoadBalancer` service using the following commands:

```

File: README.md [hunk 48]
```diff
@@ -1,29 +1,330 @@
...
+
+⚠️ Change the password before, and let's forward HTTP to HTTPS with a self-signed certificate.
+
+Change istio-ingressgateway to LoadBalancer
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

File: README.md [hunk 49]
```diff
@@ -1,29 +1,330 @@
...
+
+    cd $HOME/keys
+    $HOME/keys/step certificate create root.cluster.local root.crt root.key --profile root-ca --no-password --insecure --kty=RSA
+
+  Get Ingress IP to generate certificate
+  
+    External_IP=$(kubectl get svc istio-ingressgateway -n istio-system -o=jsonpath="{.status.loadBalancer.ingress[0].ip}")
+
+    cd $HOME/keys
+    $HOME/keys/step certificate create $External_IP.nip.io tls-$External_IP.crt tls-$External_IP.key --profile leaf  --not-after 8760h --no-password --insecure --kty=RSA --ca $HOME/keys/root.crt --ca-key $HOME/keys/root.key
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

Please replace this content with the content from here: https://gist.github.com/Djelibeybi/52ff365ef09e19a05c9df37cfca34b13 which does not rely on any third-party tool to generate certificates.

Comment (Djelibeybi) [reply README.md]: This is also available as a script that will generate the certificates for you: https://gist.github.com/Djelibeybi/0ad564baa715045dc9feea69e7113086

File: README.md [hunk 50]
```diff
@@ -1,29 +1,330 @@
...
+
+  * Create Kubernetes Secret TLS Certificate
+
+        cd $HOME/keys
+        kubectl create secret tls kubeflow-tls-cert --key=tls-$External_IP.key --cert=tls-$External_IP.crt -n istio-system
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
  * Create a Kubernetes Secret to store the certificate

        cd $HOME/kubeflow-ssl
        kubectl create secret tls kubeflow-tls-cert --key=$DOMAIN.key --cert=$DOMAIN.crt -n istio-system    
```

File: README.md [hunk 51]
```diff
@@ -1,29 +1,330 @@
...
+
+    cd $HOME/keys
+    $HOME/keys/step certificate create $External_IP.nip.io tls-$External_IP.crt tls-$External_IP.key --profile leaf  --not-after 8760h --no-password --insecure --kty=RSA --ca $HOME/keys/root.crt --ca-key $HOME/keys/root.key
+
+  * Create Kubernetes Secret TLS Certificate
+
+        cd $HOME/keys
+        kubectl create secret tls kubeflow-tls-cert --key=tls-$External_IP.key --cert=tls-$External_IP.crt -n istio-system    
+
+  * Update Kubeflow API Gateway
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
  * Update the Kubeflow API Gateway definition
```

File: README.md [hunk 52]
```diff
@@ -1,29 +1,330 @@
...
+        kind: List
+        metadata:
+          resourceVersion: ""
+          selfLink: ""
+        EOF
+
+        kubectl apply -f $HOME/kubeflow_1.6/sslenableingress.yaml
+        kubectl get gateway -n kubeflow
+
+Congratulations you installed Kubelow on OKE.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
Congratulations! You have successfully installed Kubeflow on your OKE cluster.
```

File: README.md [hunk 53]
```diff
@@ -1,29 +1,330 @@
...
+
+Access $External_IP.nip.io
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
Visit https://<Load Balancer External IP>.nip.io to access the Kubeflow Dashboard.
```

File: README.md [hunk 54]
```diff
@@ -1,29 +1,330 @@
...
+
+<!-- > Make sure Security List or NSG give access from outside LB TCP/80 and TCP/443 -->
+
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

File: README.md [hunk 55]
```diff
@@ -1,29 +1,330 @@
...
+![Access Kubeflow Dashboard](images/AccessKF.png)
+
+<!-- ## Notes/Issues
+MISSING -->
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
```

File: README.md [hunk 56]
```diff
@@ -1,29 +1,330 @@
...
 
 ## URLs
-* Nothing at this time
+
+www.kubeflow.org
```
Comment (Djelibeybi) [inline README.md]:
```suggestion

Kubeflow: <https://www.kubeflow.org/>
```

File: README.md [hunk 57]
```diff
@@ -1,29 +1,330 @@
...
 
 ## Contributing
+
 This project is open source.  Please submit your contributions by forking this repository and submitting a pull request!  Oracle appreciates any contributions that are made by the open source community.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
Oracle appreciates any contributions that are made by the community. Please submit your contributions by forking this repository and submitting a pull request. Opening an issue to discuss your contribution beforehand, especially if it's a new feature, would be appreciated.
```

File: README.md [hunk 58]
```diff
@@ -1,29 +1,330 @@
...
 
 ## License
-Copyright (c) 2021 Oracle and/or its affiliates.
+
+Copyright (c) 2022 Oracle and/or its affiliates.
 
 Licensed under the Universal Permissive License (UPL), Version 1.0.
```
Comment (Djelibeybi) [inline README.md]:
```suggestion
Copyright (c) 2022 Oracle and/or its affiliates.

Licensed under the Universal Permissive License (UPL), Version 1.0 as shown at https://opensource.oracle.com/licenses/upl/
```

**Review by Djelibeybi — CHANGES_REQUESTED** [review]
Here is my review. The only thing I strongly encourage is removing any use of third-party tools for security-related stuff, so I took out that bit and provided a script and text instead, that use OpenSSL which is available in Cloud Shell. 

Also, it's considered bad practice to commit commented out code. If it's not needed, remove it. If you need to reference it in future, that's what `git` is for.

Where possible, I've submitted my review as suggested changes, but please triple check the formatting of the commit if you chose to click GitHub's buttons, as it can and does mangle Markdown in Markdown.

**Comment by Djelibeybi** [comment]
BTW, I encourage you to use https://github.com/markdownlint/markdownlint to check your Markdown syntax. There is a Visual Studio Code plugin that embeds it to make it really easy to use.

**Review by snafuz — APPROVED** [review]

**Pull request merged**